SteamProject/MySteam/Category.cpp

```diff
@@ -33,24 +62,43 @@ bool FCategory::AddGame(const FGame NewGame)
     return false;
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
 bool FCategory::HasSpaceForGames() const
 {
     return GamesAmount < MaxGamesAmount;
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
 bool FCategory::IsValid() const
 {
     return !CategoryName.empty();
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
 void FCategory::ShowAllGames() const
 {
-    for (FGame LoopGame : GameArray)
+    for (const FGame& LoopGame : GameArray)
```

Comment:
This name is explicit in some sense, but maybe not in the correct way.

You shouldn't need to specify that this is the game in the loop since that should be clear in context. A good technique is to read your code out loud.

What makes more sense?
"For every game in the game array" or "for every loop-game in the game array"?